targets/riscv64.json

```diff
@@ -0,0 +1,13 @@
+{
+	"inherits": ["riscv"],
+	"llvm-target": "riscv64--none",
+	"build-tags": ["tinygo.riscv64"],
+	"cflags": [
+		"--target=riscv64--none",
+		"-march=rv64gc",
```

Comment:
Assuming RV64GC is a big assumption but I'm fine with it for now (it can always be changed). The riscv32.json makes a similar assumption.